Lib/test/support/__init__.py

```diff
@@ -108,6 +108,18 @@
     "run_with_tz", "PGO", "missing_compiler_executable", "fd_count",
     ]
 
+
+# True if Python is running on Microsoft Windows.
+MS_WINDOWS = (sys.platform == 'win32')
```

Comment:
Are we happy this is suitable for replacing all of the following without changing behaviour?
```python
mswindows = (sys.platform == "win32")
MS_WINDOWS = (sys.platform == 'win32')
MS_WINDOWS = (os.name == 'nt')
WIN32 = (sys.platform == "win32")
sys.platform[:3] == 'win'
```

Reply:
> sys.platform[:3] == 'win'

On "win64", Python uses "win32" as well.

> MS_WINDOWS = (os.name == 'nt')

I'm not sure about Cygwin which is a strange beast.